### DISCUSSION TITLE
Top source AS, Top source ports, Top protocols, Top source countries and  IPv4/IPv6 empty in main dashboard

### DISCUSSION THREAD

**Discussion by ABEKE8** [akvorado/akvorado#1440] [Q&A]
Hi everyone, I  have this problem, don't solved after i add config sampling output. Thanks and appreciate your feedback
![Screenshot from 2024-10-09 09-09-41](https://github.com/user-attachments/assets/b4524f7a-66a0-4a76-98fa-416667552b7a)

**Comment by ABEKE8** [comment]
and I removed the filter.

**Reply by Tayyab146** [reply]
Did it work?

**Reply by ABEKE8** [reply]
yes ,  go to >>>

Traffic not display on dashboard but can see flow received? #1424

**Comment by joseluissp** [comment]
Have you solved?

**Reply by ABEKE8** [reply]
yes ,  go to >>>

Traffic not display on dashboard but can see flow received? #1424
